src/main/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/rca/configs/decider/jvm/LevelTwoActionBuilderConfig.java

```diff
@@ -0,0 +1,48 @@
+package com.amazon.opendistro.elasticsearch.performanceanalyzer.rca.configs.decider.jvm;
+
+import com.amazon.opendistro.elasticsearch.performanceanalyzer.rca.configs.ConfigUtils;
+import java.util.Map;
+
+public class LevelTwoActionBuilderConfig {
```

Comment:
Can these be merged with queue/cache action configs?

Reply:
this config object currently defines the step size of each action that decider is going to issue in each level. and we might add more knobs for each level in the future. so i think it would be better to keep this config separately